site/docs/v1.4/resource-filtering.md

```diff
@@ -0,0 +1,129 @@
+# Resource Filtering
+
+*Filter objects by namespace, type, or labels.*
+
+Velero includes all objects in a backup or restore when no filtering options are used. 
+
+## Includes
+
+Only specific resources are included, excluding all others.
+
+Wildcard takes precedence when both a wildcard and specific resource are included.
+
+### --include-namespaces
+
+* Backup a namespace and it's objects.
+
+  ```bash
+  velero backup create <backup-name> --include-namespaces <namespace>
+  ```
+
+* Restore two namespaces and their objects.
+
+  ```bash
+  velero restore create <backup-name> --include-namespaces <namespace1>,<namcespace2>
+  ```
+
+### --include-resources 
+
+* Backup all deployments in the cluster.
+
+  ```bash
+  velero backup create <backup-name> --include-resources deployments
+  ```
+
+* Restore all deployments and configmaps in the cluster.
+
+  ```bash
+  velero restore create <backup-name> --include-resources deployments,configmaps
+  ```
+
+* Backup the deployments in a namespace.
+
+  ```bash
+  velero backup create <backup-name> --include-resources deployments --include-namespaces <namespace>
+  ```
+
+### --include-cluster-resources
+
+  This option can have three possible values:
+
+* `true`: all cluster-scoped resources are included.
+
+* `false`: no cluster-scoped resources are included.
+
+* `nil` ("auto" or not supplied):
+
+  - Cluster-scoped resources are included when backing up or restoring all namespaces. Default: `true`.
+
+  - Cluster-scoped resources are not included when namespace filtering is used. Default: `false`.
+
+    * Some related cluster-scoped resources may still be backed/restored up if triggered by a custom action (e.g. PVC->PV) unless `--include-cluster-resources=false`.
```

Comment:
Referring to [our new style guide](https://velero.io/docs/main/style-guide/#avoid-latin-phrases), let's use `for example` instead of `e.g.`.

Reply:
Done.